Schedules.API/Tasks/Sending/SendSMS.cs

```diff
@@ -17,15 +17,19 @@ public override void Execute ()
       var token = EnvironmentVariableHelper.GetEnvironmentVariable(tokenKey);
       var number = EnvironmentVariableHelper.GetEnvironmentVariable(numberKey);
 
+
```

Comment:
Please remove the changes from this file, it's not specific to documentation changes